cpu/msp430fxyz/msp430_stdio.c

```diff
@@ -40,3 +43,17 @@ int putchar(int c)
     uart_stdio_write(&_c, 1);
     return 1;
 }
+
+/**
+ * @brief   Write nbyte characters to the STDIO UART interface
+ */
+ssize_t write(int fildes, const void *buf, size_t nbyte)
+{
+    if (fildes == STDOUT_FILENO) {
+        uart_stdio_write(buf, nbyte);
```

Comment:
```uart_stdio_write()``` doesn't return anything, as writing to uart doesn't fail on any of our platforms.